sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyVaultKey.java

```diff
@@ -60,13 +63,29 @@ public KeyProperties getProperties() {
         return this.properties;
     }
 
+    /**
+     * Get the key type of the key
+     * @return the key type
+     */
+    public KeyType getKeyType() {
+        return key.getKeyType();
+    }
+
+    /**
+     * Get the key operations of the key
+     * @return the key operations
+     */
+    public List<KeyOperation> getKeyOperations() {
+        return key.getKeyOps();
+    }
```

Comment:
Are you ok with users modifying the key operations list here? Should this be a clone?

Reply:
Made it unmodifiable list